.gitignore

```diff
@@ -1,71 +1,74 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-
-# C extensions
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
 *.so
-
-# Distribution / packaging
-.Python
-env/
+*.pyc
 build/
-develop-eggs/
 dist/
-downloads/
-eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-*.egg-info/
-.installed.cfg
-*.egg
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
+doc/build/
 
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.coverage
-.cache
-nosetests.xml
-coverage.xml
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
 
-# Translations
-*.mo
-*.pot
-
-# Django stuff:
+# Logs and databases #
+######################
 *.log
+*.sql
+*.sqlite
+logs/*
 
-# Sphinx documentation
-docs/_build/
```

Comment:
This is needed if we are going to continue using sphinx.

Reply:
++ I'll add this to a task for a follow on commit.

Reply:
done